### ISSUE TITLE
ULS UNITS ASIA - Inline links need to meet ratio and be underlined

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#454]
Inline links fail the 3:1 difference (1.3:1).  Inline links need to be underlined
